repo2docker/docker.py

```diff
@@ -141,12 +141,18 @@ def build(
 
                 args += [d]
 
-                yield from execute_cmd(args, True)
+                for line in execute_cmd(args, True):
+                    # Simulate structured JSON output from buildx build, since we
+                    # do get structured json output from pushing and running
+                    yield {"stream": line}
```

Comment:
Is `line` guaranteed to never contain characters that need to be escaped?

Reply:
@manics not that I know of! It's also fundamentally the same process as what we were getting from the python client, just through subprocess instead.

Reply:
I'll try out a bunch and keep an eye on it though!